src/Components/Reviews.jsx

```diff
@@ -0,0 +1,36 @@
+import { getReviews } from "../api"
+import { useEffect, useState } from 'react'
+
+const Reviews = () => {
+    const [reviewList, setReviews] = useState([])
+
+    useEffect(()=>{
+        getReviews()
+        .then((reviews)=>{
+            setReviews(reviews)
+        })
+    }, [reviewList])
+    return (
+        <div>
+            <h2> All Reviews </h2>
+            <div className="reviews">
+            <ol>
+                {reviewList.map((review)=>{
+                    return (
+                        <li className="review" key={review.review_id}>
+                            <h3>{review.title}</h3>
+                            <br></br>
```

Comment:
This looks good, I'd propose maybe adding the author name too? That would be a nice summary with just enough but not too much info on the article... showing the user the article title, author name and an image